Integrations/MimecastV2/MimecastV2.py

```diff
@@ -1479,6 +1471,605 @@ def download_attachment_request(attachment_id):
+    }
+
+    response = http_request('POST', api_endpoint, str(payload))
+    if isinstance(response, dict) and response.get('fail'):
+        return_error(json.dumps(response.get('fail', [{}])[0].get('errors')))
+    return response
+
+
+def get_mimecast_incident():
+    api_response = get_mimecast_incident_request()
+
+    markdown_output = mimecast_incident_api_response_to_markdown(api_response, 'get')
+    entry_context = mimecast_incident_api_response_to_context(api_response)
+
+    return_outputs(markdown_output, entry_context, api_response)
+
+
+def get_mimecast_incident_request():
+    api_endpoint = '/api/ttp/remediation/get-incident'
+    incident_id = demisto.args().get('incident_id', '').encode('utf-8')
+
+    data = {
+        'id': incident_id
+    }
+
+    payload = {
+        'data': [data]
+    }
+
+    response = http_request('POST', api_endpoint, str(payload))
+    if isinstance(response, dict) and response.get('fail'):
+        return_error(json.dumps(response.get('fail', [{}])[0].get('errors')))
+    return response
+
+
+def mimecast_incident_api_response_to_markdown(api_response, action_type):
+    incident_code = api_response['data'][0]['code']
+    incident_type = api_response['data'][0]['type']
+    incident_reason = api_response['data'][0]['reason']
+    incident_identified_messages_amount = api_response['data'][0]['Identified']
+    incident_successful_messages_amount = api_response['data'][0]['Successful']
+    incident_failed_messages_amount = api_response['data'][0]['Failed']
+    incident_restored_messages_amount = api_response['data'][0]['Restored']
+    incident_id = api_response['data'][0]['id']
+
+    if action_type == 'create':
+        md = 'Incident ' + incident_id + ' has been created\n'
+    else:
+        md = 'Incident ' + incident_id + ' has been found\n'
+    md += '\n####Code: ' + incident_code
+    md += '\n####Type: ' + incident_type
+    md += '\n####Reason: ' + incident_reason
+    md += '\n####The number of messages identified based on the search criteria: ' + incident_identified_messages_amount
+    md += '\n####The number successfully remediated messages: ' + incident_successful_messages_amount
+    md += '\n####The number of messages that failed to remediate: ' + incident_failed_messages_amount
+    md += '\n####The number of messages that were restored from the incident: ' + incident_restored_messages_amount
+
+    messages_table_list = list()
+    for message in api_response['data'][0]['searchCriteria']:
+        message_entry = {
+            'From': message['from'],
+            'To': message['to'],
+            'Start date': datetime.strptime(message['start'], '%Y-%m-%dT%H:%M:%SZ'),
+            'End date': datetime.strptime(message['end'], '%Y-%m-%dT%H:%M:%SZ'),
+            'Message ID': message['messageId'],
+            'File hash': message['fileHash']
+        }
+
+        messages_table_list.append(message_entry)
+
+    md = tableToMarkdown(md, messages_table_list,
```

Comment:
seems like a really long table header name

Reply:
Done in all relevant places.